### PR TITLE
Feature/gh 5110 move prev next btns to slot

### PR DESCRIPTION
**Pull request by steinsebastian**

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts [hunk 1]
```diff
@@ -1,18 +1,137 @@
...
     TestBed.configureTestingModule({
       imports: [I18nTestingModule],
       declarations: [ConfigPreviousNextButtonsComponent],
+      providers: [
+        {
+          provide: RoutingService,
+          useClass: MockRoutingService,
+        },
+
+        {
```
Comment (schuppell) [inline projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts]: ConfiguratorCommonsService not used.

File: projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts [hunk 2]
```diff
@@ -1,18 +1,137 @@
...
+  },
+};
+
+class MockRoutingService {
+  getRouterState(): Observable<RouterState> {
+    return of(mockRouterState);
+  }
+}
+
+class MockConfiguratorCommonsService {
```
Comment (schuppell) [inline projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts]: Remove unneeded Mock class

File: projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.ts [hunk 3]
```diff
@@ -2,29 +2,107 @@ import {
...
+      this.routingService
+        .getRouterState()
+        .subscribe(state => this.initConfigurationGroups(state))
+    );
+  }
+
+  initConfigurationGroups(routingData) {
+    this.productCode = routingData.state.params.rootProduct;
+
+    this.configuration$ = this.configuratorCommonsService.getConfiguration(
```
Comment (schuppell) [inline projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.ts]: Not needed.

File: projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts (hunk 2 above)
Comment (steinsebastian) [reply projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.spec.ts]: done

File: projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.ts (hunk 3 above)
Comment (steinsebastian) [reply projects/storefrontlib/src/cms-components/configurator/commons/config-previous-next-buttons/config-previous-next-buttons.component.ts]: Done

**Pull request merged**